presto-docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -251,14 +251,16 @@ Property Name                             Description
 ``location``                               Optionally specifies the file system location URI for
                                            the table.
 
-``format_version``                         Optionally specifies the format version of the Iceberg
+``format_version``                         Optionally specifies the format version of the Iceberg           ``2``
```

Comment:
@steveburnett, I have made changes to the default format version and resolved other merge conflicts. Please review when you get a chance, thanks!